examples/quickstart/main.rs

```diff
@@ -190,7 +190,9 @@ async fn main() {
                 user_address.clone(),
             );
 
-            print!("Do you want to simulate, execute or skip this swap? (simulate/execute/skip): ");
+            println!("Do you want to simulate, execute or skip this swap?");
+            println!("Please be aware that the market might move while you make your decision.");
```

Comment:
"...leading to a revert if you've set a min amount out or slippage."

Unless they will see the revert message nicely if it reverts?

Reply:
no, they don't see a nice message 🙃